packages/extension-api-explorer/package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "@looker/extension-api-explorer",
-  "version": "1.0.0",
+  "version": "1.0.4-alpha.0",
```

Comment:
What do you think about changing the extension version to match that of `@looker/api-explorer`?

Reply:
It would be attractive to version all the packages via lerna, or get rid of `-alpha`. Bigger issue than just this PR.